process/coordinator/process.go

```diff
@@ -719,6 +719,11 @@ func (tc *transactionCoordinator) CreateMbsAndProcessCrossShardTransactionsDstMe
 				"total gas penalized", tc.gasHandler.TotalGasPenalized(),
 				"error", errProc,
 			)
+
+			if core.IsGetNodeFromDBError(errProc) {
+				return nil, 0, false, err
```

Comment:
unit test?

Reply:
Added. Also, return `errProc` instead of `err`

Reply:
Actually, removed this check. As discussed, it is not needed here.